lib/msal-common/src/config/Configuration.ts

```diff
@@ -21,13 +21,26 @@ const DEFAULT_TOKEN_RENEWAL_OFFSET_SEC = 300;
  * - crypto: implementation of crypto functions
  */
 export type Configuration = {
+    authOptions?: AuthOptions,
     systemOptions?: SystemOptions,
     loggerOptions?: LoggerOptions,
     storageInterface?: ICacheStorage,
     networkInterface?: INetworkModule,
     cryptoInterface?: ICrypto
 };
 
+/**
+ * @type AuthOptions: Use this to configure the auth options in the Configuration object
+ *
+ *  - clientId                    - Client ID of your app registered with our Application registration portal : https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredAppsPreview in Microsoft Identity Platform
+ *  - authority                   - You can configure a specific authority, defaults to " " or "https://login.microsoftonline.com/common"
+ *  - redirectUri                 - The redirect URI of the application, this should be same as the value in the application registration portal.Defaults to `window.location.href`.
+ */
+export type AuthOptions = {
```

Comment:
Are we not allowing setting redirectUri anymore? navigateToLoginRequestUrl?

Reply:
t does. We made a choice of making `redirectUri a mandatory param in the `request`, hence removed it from `configuration`. msal-node does not need `navigateToLoginRequestUrl` as it doesn't need a browser

Reply:
```redirectUri``` is also in the block comment above. So if we're removing it from the ```AuthOptions``` we should remove it from the comment as well.

Reply:
Either that or make NodeAuthOptions and SPAAuthOptions separate types with no base type.

Reply:
Sure, I think making NodeAuthOptions and SpaAuthOptions into separate types makes sense. Having redirectUri in the base class pollutes the public API for node, as it's only used in one (out of 6+ flows).